src/index.js

```diff
@@ -649,9 +649,9 @@ export default class OfflinePlugin {
     }
 
     if (this.relativePaths) {
-      tool.location = tool.output;
+      tool.location = `__webpack_public_path__ + ${JSON.stringify(tool.output)}`;
```

Comment:
`+` sign here seems like a bug, it's in template literal already

Reply:
that is evaluated at runtime. it interpolates at build time to `__webpack_public_path__ + "some-output-file.js"`